gulpfile.js

```diff
@@ -0,0 +1,61 @@
+/**
+ *  Give Gulp File
+ *
+ *  Used for automating development tasks.
+ */
+
+/* Modules (Can be installed with npm install command using package.json)
+ ------------------------------------- */
+var gulp = require('gulp'),
+    sort = require('gulp-sort'),
+    wpPot = require('gulp-wp-pot'),
+    checktextdomain = require('gulp-checktextdomain');
+
+/* POT file task
+ ------------------------------------- */
+gulp.task('pot', function () {
+    return gulp.src('**/*.php')
+        .pipe(sort())
+        .pipe(wpPot({
+            package: 'TGM Plugin Activation',
```

Comment:
Please put these TGMPA-specific values into `package.json`, and have this file read from there.